cmd/api/controller/controller.go

```diff
@@ -45,9 +91,17 @@ func (c *PessoaController) Get(ctx *fiber.Ctx) error {
 func (c *PessoaController) GetByTerm(ctx *fiber.Ctx) error {
 	pessoaTerm := ctx.Query("t")
 
+	if pessoaTerm == "" {
+		return fiber.NewError(http.StatusBadRequest, "invalid query param")
+	}
+
 	pessoas, err := c.service.GetPessoaByTerm(pessoaTerm)
 	if err != nil {
-		return err
+		return fiber.NewError(http.StatusInternalServerError, ErrInvalidJson.Error())
+	}
+
+	if pessoas == nil {
+		return ctx.Status(fiber.StatusBadRequest).JSON(ErrNotFound.Error())
 	}
```

Comment:
Parece estranho isso ser um bad request, já que pessoas é retornado pelo serviço, isso não seria um `fiber.StatusNotFound`?

```suggestion
	if pessoas == nil {
		return ctx.Status(fiber.StatusNotFound).JSON(ErrNotFound.Error())
	}
```

Reply:
Então, neste caso a regra de negócio pediu que retornasse bad request.

Eu havia implementado como Not Found, mas numa repassagem eu percebi este ponto e fiz conforme a regra